sdks/python/apache_beam/runners/interactive/pipeline_instrument.py

```diff
@@ -595,6 +596,10 @@ def _process(self, pcoll):
       self._cache_manager = ie.current_env().get_cache_manager(
           self._user_pipeline, create_if_absent=True)
 
+  def _is_streaming_pcollection(self, pcoll):
```

Comment:
This should be called _is_globally_windowed. 

Whether or not something is streaming is orthogonal to what windowing it is in.

Reply:
An even better test might be `pcoll.windowing.is_default()`.